middleware/helpers.js

```diff
@@ -50,3 +50,35 @@ module.exports.getMaxDate = () => {
 
   return `${year}-${month}-${day}`;
 };
+
+function checkMediaLink(mediaLink, linkRegex) {
+  const match = mediaLink.match(linkRegex);
+  return match && match[3];
+}
+
+module.exports.mediaLinkValidation = (facebook, twitter, instagram, tumblr) => {
+  const mediaRegex = {
+    facebook:
+      /^(https?:\/\/)?(www\.)?facebook\.com\/(?:[a-zA-Z0-9.]+\/)?(?:profile\.php\?id=(\d+)|([a-zA-Z0-9.]+))\/?$/,
+    twitter: /^(https?:\/\/)?(www\.)?twitter\.com\/([a-zA-Z0-9_]{1,15})\/?$/,
+    instagram:
+      /^(https?:\/\/)?(www\.)?instagram\.com\/([a-zA-Z0-9_.]{1,30})\/?$/,
+    tumblr: /^(https?:\/\/)?(www\.)?tumblr\.com\/([a-zA-Z0-9_.]{1,30})\/?$/,
+  };
+  let validLinks = {};
```

Comment:
Use `const` keyword if you're not planning on overriding the value of `validLinks`